packages/loader/container-loader/src/container.ts

```diff
@@ -1730,6 +1732,15 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         // are set. Global requests will still go directly to the loader
         const loader = new RelativeLoader(this.loader, () => this.originalRequest);
         const previousCodeDetails = this._context?.codeDetails;
+        const loadContainerCopyFn = async (additionalHeaders: IRequestHeader) => {
+            // Load the container copy restricted: uncached and no reconnect
+            return this.loadContainerCopy(
+                this.loader,
+                this.loader.services.urlResolver,
+                this.originalRequest?.url,
```

Comment:
I removed originalRequest in recent main, you'll want to merge it
```suggestion
                this.containerUrl,
```